admin/AppModel.js

```diff
@@ -14,11 +14,10 @@ import {loggingTab} from './tabs/logging/LoggingTab';
 import {monitorTab} from './tabs/monitor/MonitorTab';
 import {userDataTab} from './tabs/userData/UserDataTab';
 
-@HoistAppModel
-export class AppModel {
+export class AppModel extends HoistAppModel {
```

Comment:
glad you brought this up.  Wanted to discuss.  So (weirdly) mobx *requires* that there be annotations on your objects if you are going to call makeObservable(this) with no second argument.  Otherwise it throws.  

We could put a dummy observable that always appears on HoistBase -- or actually just stop making the loadSupport related observables dynamic.  Think the latter is probably the right thing to do. 

also need to make triple-sure that its ok in mobx to be re-registering these observables twice, and calling makeObservable multiple times in the constructor chain